test/FsAutoComplete.IntegrationTests/TestHelpers.fsx

```diff
@@ -80,9 +80,9 @@ let testConfig =
       else
         match fsacRuntime with
         | FSACRuntime.NETCoreFDD ->
-          fromRoot "../../src/FsAutoComplete/bin/%s/%s/fsautocomplete.dll" configuration framework
+          fromRoot "../../src/FsAutoComplete/bin/%s/netcoreapp2.1/fsautocomplete.dll" configuration
```

Comment:
hardcoded these because there's a distinction between the FSAC runtime (netcoreapp2.1) and the fsi execution runtime (netcoreapp3.0) that was hard to overcome given the current fsi test model we have.